shaarli_client/utils.py

```diff
@@ -21,3 +22,27 @@ def generate_all_endpoints_parsers(subparsers, endpoints):
     """Generate all endpoints' subparsers from an endpoints dict"""
     for ep_name, ep_metadata in endpoints.items():
         generate_endpoint_parser(subparsers, ep_name, ep_metadata)
+
+
+def write_output(args, response):
```

Comment:
@virtualtam Let me know if anything should be added. Do I need to add some tests, and what should I test for?